pom.xml

```diff
@@ -1253,6 +1253,38 @@
                 <module>assembly</module>
             </modules>
         </profile>
+        <profile>
+            <id>cdh5.14.2</id>
+            <activation>
+                <property>
+                    <name>env</name>
+                    <value>cdh5.14.2</value>
+                </property>
+            </activation>
+            <properties>
+                <cdh.parcel.version>5.14.2</cdh.parcel.version>
+                <envClassifier>cdh5.14.2</envClassifier>
+                <envHbase>hbase1.2.0</envHbase>
+                <hadoop.version>2.6.0-cdh5.14.2</hadoop.version>
+                <hbase.version>1.2.0-cdh5.14.2</hbase.version>
+                <hive.version>1.1.0-cdh5.14.2</hive.version>
+                <spark.major.version>spark2.2</spark.major.version>
+                <zookeeper.version>3.4.5-cdh5.14.2</zookeeper.version>
+                <spark.version>2.2.0.cloudera2</spark.version>
+                <spark-assembly-id>spark-assembly-hadoop2.6.0-cdh5.14.2</spark-assembly-id>
+                <kafka.version>0.10.2-kafka-2.2.0</kafka.version>
```

Comment:
0.10.0-kafka-2.1.0

Reply:
For the branch-2.7, i'm not sure for the reason, but the kafka version is different from master (0.10.2-kafka-2.2.0 is a more recent version than 0.10.0-kafka-2.1.0). 
Above this shows 
```
<kafka.version>0.10.2-kafka-2.2.0</kafka.version>
```
For the other 9 cloudera platforms. Should master match/be updated?

Reply:
I believe kafka.version is currently irrelevant in 2.7. I had to downgrade it when I added Spark 2.3 support. If we backport it to 2.7 we'll need to use the correct version. I still think it's better to set it right for 5.14.2 now as it won't be part of the Spark2.3 backport.